rust/connlib/libs/gateway/src/control.rs

```diff
@@ -144,13 +144,15 @@ impl<CB: Callbacks + 'static> ControlPlane<CB> {
 impl<CB: Callbacks + 'static> ControlSession<IngressMessages, CB> for ControlPlane<CB> {
     #[tracing::instrument(level = "trace", skip(private_key, callbacks))]
     async fn start(
+        fd: Option<i32>,
```

Comment:
We are going to change it later so it doesn't matter but I think it's better if we expect here a `RawFd`.

Reply:
Yeah, this thought was already crossing my mind as I wrote this, so I'll note it down for when we change it!